### ISSUE TITLE
Screen blinks to white after opening

### ISSUE THREAD

**Issue by mergen3107** [koreader/koreader#5954]
* KOReader version: 2020.03.01
* Device: Kindle PW3 (5.9.7)

#### Issue
Sometimes after opening KOReader the screen goes completely white. However, everything seems to be in place at this time, because I can follow links by tapping them blindly if they are present.
Happened with fb2.zip, doc, djvu, pdf format. Seems like it is format-independent. 
As if there is one extra round of screen refresh which forgets to draw everything back.

#### Steps to reproduce
Open a book from Kindle menu (with KPVBooklet 0.6.6) or from KUAL menu.
The book shows up, everything looks OK.
Wait for about 10-15 seconds - the screen blinks once and stays white.
Then if I turn a page, everything comes back.
I blindly tapped on a link that was on that page, but because it was a link from Facebook wrapper, KOReader complained that the link is invalid and showed a window. After I closed the window, only the area under this window was redrawn.

**Comment by NiLuJe** [comment]
Known issue w/ KPVBooklet.

Shouldn't happen w/ KUAL, though.

**Comment by NiLuJe** [comment]
Note that having used KPV booklet once in that power cycle may trigger the issue on subsequent KUAL launches, too.

**Comment by mergen3107** [comment]
Thank you for the info!

**Comment by mergen3107** [comment]
Will restarting Kindle resolve this issue until I again use KPVBooklet?

**Comment by mergen3107** [comment]
Is there another way to quickly launch KOReader? Like a KUAL file, but for KOReader directly?
I remember that since one FW version we can't replace Kindle menu's store button anymore?

**Comment by NiLuJe** [comment]
> Will restarting Kindle resolve this issue until I again use KPVBooklet?

Yep, things will start to go wonky after the first or second launch through KPVBooklet, until the next full reboot.

**Comment by NiLuJe** [comment]
> Is there another way to quickly launch KOReader? Like a KUAL file, but for KOReader directly?
> I remember that since one FW version we can't replace Kindle menu's store button anymore?

Not really. You can write a custom KUAL extension to put a KOReader enbtry somewhere on the first top-level page instead of in a submenu, but that's it.

**Comment by mergen3107** [comment]
OK! Thank you! :)

**Comment by mergen3107** [comment]
So I uninstalled KPVBooklet and since then I can't catch another issue I was getting before.
The issue was that after pressing power button KOReader drew two screensavers one after another consistently. Screensaver folder is /mnt/us/linkss/screensavers from your ScreensaverHack, and both screensavers were taken from it.
Is this issue related to KPVBooklet?
If yes, should we rename this issue title?

**Comment by NiLuJe** [comment]
No, that probably won't happen again after #5943 (and/or it'll break it even more ^^).

But, yes, that's another KPVBooklet-specific potential quirk.
